packages/react/src/forms/TextInput/TextInput.module.css

```diff
@@ -83,9 +83,14 @@
 }
 
 .TextInput-wrapper--fullWidth {
+  width: 100%;
```

Comment:
If we do this, we'd need a changeset for this. Related comment [here](https://github.com/primer/brand/pull/820/files#r1851823899).

Reply:
Splitting this fix out into its own PR

https://github.com/primer/brand/pull/828

I'll create an issue to track this shortly